src/main/java/org/gridsuite/directory/server/DirectoryService.java

```diff
@@ -31,6 +32,7 @@
 
 import static org.gridsuite.directory.server.DirectoryException.Type.NOT_ALLOWED;
 import static org.gridsuite.directory.server.DirectoryException.Type.NOT_FOUND;
+import static org.gridsuite.directory.server.NotificationType.UPDATE_DIRECTORY;
```

Comment:
Maybe remove this static import and use NotificationType.UPDATE_DIRECTORY in code.

Reply:
Done